File: set1 q1/solution.py
def find_maximum_element(arr):
    if not arr:
        return None
    # if inbuilt functions can be used
    # return max(arr) 

    # without using inbuilt functions
    max_element = arr[0]
    for num in arr:
        if num > max_element:
            max_element = num
    return max_element

array = [3, 5, 90, 8, 4]
print("The maximum element is:", find_maximum_element(array))

# output is The maximum element is: 90 which is correct

File: set1 q1/solution.js
function findMaximumElement(arr) {
    if (arr.length === 0) {
        return null;
    }

    // if inbuilt functions can be used
    // return Math.max(...arr);

    // without using inbuilt functions
    let maxElement = arr[0];
    for (let i = 1; i < arr.length; i++) {
        if (arr[i] > maxElement) {
            maxElement = arr[i];
        }
    }
    return maxElement;
}



const array = [3, 5, 1, 8, 4];
console.log("The maximum element is:", findMaximumElement(array));

// out put is The maximum element is: 8 which is correct